source/deployment/sso-saml.rst

```diff
@@ -1,12 +1,16 @@
-==========================
+=========================
 SAML Single Sign-On (E20)
-==========================
+=========================
+
+Single sign-on (SSO) is a way for users to log into multiple applications with a single user ID and password without having to re-enter their credentials. The SAML standard allows identity providers to pass credentials to service providers. Mattermost can be configured to act as a SAML 2.0 Service Provider. 
+
+The SAML single-sign-on integration offers the following benefits:
 
 Mattermost can be configured to act as a SAML 2.0 Service Provider. The SAML Single sign-on integration offers the following benefits:
```

Comment:
Recommend removing this line - redundant to above.

Reply:
Removed.